test/functional/test_framework/test_node.py

```diff
@@ -557,9 +557,9 @@ def add_p2p_connection(self, p2p_conn, *, wait_for_verack=True, **kwargs):
         return p2p_conn
 
     def add_outbound_p2p_connection(self, p2p_conn, *, p2p_idx, connection_type="outbound-full-relay", **kwargs):
-        """Add an outbound p2p connection from node. Either
-        full-relay("outbound-full-relay") or
-        block-relay-only("block-relay-only") connection.
+        """Add an outbound p2p connection from node. Must be a
+        full-relay("outbound-full-relay"), block-relay-only("block-relay-only")
```

Comment:
```suggestion
        full-relay ("outbound-full-relay"), block-relay-only ("block-relay-only")
```
or: `Must be an "outbound-full-relay", "block-relay-only" or "addr-fetch" connection.`

Reply:
Took the short version, that should be clear enough.